pkg/ebpftracer/tracer_decode.go

```diff
@@ -59,6 +62,85 @@ func (t *Tracer) decodeAndHandleSignal(_ context.Context, data []byte) (rerr err
 		if err != nil {
 			return fmt.Errorf("cannot remove cgroup %d from mute map: %w", args.CgroupId, err)
 		}
+
+	case types.SignalSchedProcessExecArgs:
```

Comment:
Not sure about this approach. This means that now we will get duplicate events from kernel for exec in signal pipeline and here.

Instead would be better to add separate priority queue for such signal events and use the same types.Context and args.